text/0005-application-formalization.md

```diff
@@ -0,0 +1,194 @@
+| metadata          | Metadata     | System-generated record metadata                                         | Read-only. See [Metadata Schema] (https://github.com/folio-org/raml/blob/master/schemas/metadata.schema) |
+
+**Dependency Properties**
+| Property | Type    | Description                                                          | Notes                            |
+| ---------| ------- | -------------------------------------------------------------------- | -------------------------------- |
+| id       | String  | Identifier for the module conforming to pattern: `{name}-{version}`  | e.g. "mod-orders-storage-13.5.0" |
+| name     | String  | Name of the module                                                   | e.g. "mod-orders"                |
+| version  | String  | Version of the module                                                | e.g. "13.5.0"                    |
+
+**ModuleId Properties**
+| Property | Type    | Description                                                          | Notes                            |
+| -------- | ------- | -------------------------------------------------------------------- | -------------------------------- |
+| id       | String  | Identifier for the module conforming to pattern: `{name}-{version}`  | e.g. "mod-orders-storage-13.5.0" |
+| name     | String  | Name of the module                                                   | e.g. "mod-orders"                |
+| version  | String  | Version of the module                                                | e.g. "13.5.0"                    |
+| url      | String  | URL pointing to the module descriptor for this module                | e.g. "https://folio-registry.dev.folio.org/_/proxy/modules/mod-orders-storage-13.5.0" |
+
+Example:
+```
+{
+  "id": "app-gobi-0.0.1",
+  "name": "app-gobi",
+  "version": "0.0.1",
+  "description": "Application facilitating the placement of orders via GOBI and other compatible vendors",
+  "dependencies": [
+    {
+      "id": "app-orders-0.0.1",
+      "name": "app-orders",
+      "version": "^0.0.1"
+    },
+    {
+      "id": "app-organizations-0.0.1",
+      "name": "app-organizations",
+      "version": "^0.0.1"
+    }
+  ],
+  "modules": [
+    {
+      "id": "mod-gobi-2.6.0",
+      "name": "mod-gobi",
+      "version": "2.6.0",
+	  "url": "https://folio-registry.dev.folio.org/_/proxy/modules/mod-gobi-2.6.0"
+    }
+  ],
+  "uiModules": [
+    {
+      "id": "folio_gobi-settings-2.0.0",
+      "name": "folio_gobi-settings",
+      "version": "2.0.0",
+	  "url": "https://folio-registry.dev.folio.org/_/proxy/modules/folio_gobi-settings-2.0.0"
+    }
+  ]
+}
+```
+**N.B.** Full module descriptors and metadata have been omitted for readability reasons.
+
+## Benefits
+* This is the first step toward breaking the monolithic release into application-specific releases
```

Comment:
This is **not** alluding to having several releases for several different purposes (like FOLIO-erm, FOLIO-for-public-libraries, ...), correct ? Maybe than we just change the wording to use singular: 'This is the first step toward breaking the monolithic release into an application-specific release" ?

Reply:
The idea is that instead of doing a single monolithic release where everything is released at once, we move toward a situation where individual applications are released on their own, independent of the other applications.  So app-orders might be released tomorrow, then a month later app-courses is released, and so on.  Does that help clarify things @inkuss?  I'm open to phrasing suggestions for how to make it more clear.

Reply:
Please LMK if it's OK to mark this thread resolved, or if further discussion is required

Reply:
@inkuss ?